core/whistle_amqp-1.0.0/src/wh_amqp_assignments.erl

```diff
@@ -149,6 +149,11 @@ handle_call({'request_float', Consumer, Broker}, _, State) ->
     {'reply', assign_or_reserve(Consumer, Broker, 'float'), State};
 handle_call({'request_sticky', Consumer, Broker}, _, State) ->
     {'reply', assign_or_reserve(Consumer, Broker, 'sticky'), State};
+handle_call({'release_handlers', Consumer}, _, State) ->
+    Pattern = #wh_amqp_assignment{consumer=Consumer, _='_'},
+    Res = release_handlers(ets:match_object(?TAB, Pattern, 1)),
```

Comment:
Why build Pattern up here? I'd just pass Consumer to release_handlers/1 and do the ets magic there since that is the meat of the functionality.